### PR TITLE
HPE ProLiant DX385 Gen10 Plus Server added

### PR DESCRIPTION
**Pull request by TaLoN1x**
HPE ProLiant DX385 Gen10 Plus Server added

### CODE REVIEW

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: HPE
+model: ProLiant DL385 Gen10 Plus v2
+slug: hpe-proliant-dl385-gen10-plus-v2
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
```
Comment (danner26) [inline device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+front_image: true
+rear_image: true
+console-ports:
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml:12]: VGA is not a console port type. Please remove this

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  - name: USB1
+    type: usb-a
+  - name: USB2
+    type: usb-a
```
Comment (danner26) [inline device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml:16]: Are these used for console access? If not, these do not belong here.

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: HPE
+model: ProLiant DX385 Gen10 Plus V2
+slug: hpe-proliant-dx385-gen10-plus-v2
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
```
Comment (danner26) [inline device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+front_image: true
+rear_image: true
+console-ports:
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml:12]: VGA is not a console port type. Please remove this

File: device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+slug: hpe-proliant-dx385-gen10-plus
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+front_image: true
+rear_image: true
+comments: '[ProLiant DX385 Gen10 Plus Server Specs](https://www.hpe.com/psnow/doc/a00056112enw)'
+console-ports:
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml:13]: VGA is not a console port type. Please remove this

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  - name: USB1
+    type: usb-a
+  - name: USB2
+    type: usb-a
```
Comment (danner26) [inline device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml:16]: Are these used for console access? If not, these do not belong here.

File: device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  - name: USB1
+    type: usb-a
+  - name: USB2
+    type: usb-a
```
Comment (danner26) [inline device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml:17]: Are these used for console access? If not, these do not belong here.

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml (hunk 6 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml]: fixed

File: device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml (hunk 8 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DX385-Gen10-Plus.yaml]: fixed

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml (hunk 7 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml]: fixed

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml (hunk 5 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml]: fixed

File: device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml (hunk 4 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DX385-Gen10-Plus-v2.yaml:7]: could not find, inconsistent information from suppliers and vendors. Don't want to gamble.

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml (hunk 3 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml]: fixed

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml (hunk 2 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml]: fixed

File: device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml (hunk 1 above)
Comment (TaLoN1x) [reply device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml:7]: could not find, inconsistent information from suppliers and vendors. Don't want to gamble.

**Review by TaLoN1x — COMMENTED** [review]
addressed issues

**Review by danner26 — APPROVED** [review]

**Pull request merged**